skabase/SKAMaster/SKAMaster/SKAMaster.py

```diff
@@ -118,7 +118,7 @@ def init_device(self):
         self._element_tel_state_device = ""
         self._element_database_device = ""
 
-        self._max_capabilities = {}
+        self._max_capabilities = {"cap1":10, "cap2":20}
```

Comment:
Commented on this previously - do not override in the real implementation.

Reply:
Removed. Commit 15fadfe addresses this.